src/render/image_atlas.js

```diff
@@ -140,4 +140,3 @@ export default class ImageAtlas {
 
 register('ImagePosition', ImagePosition);
 register('ImageAtlas', ImageAtlas);
-
```

Comment:
nit: no change needed

Reply:
This was automatically added by my editor. It's pretty standard to end files with a new line isn't it? Most of our other files have a trailing new line.